File: wlada-si.21.19.getout-e707bbd07835/Aplikacija/SWEproj/frontend/src/app/models/korisnik.ts

export class Korisnik
{
    public id: number;
    public uloga: string;
    public ime: string;
    public prezime: string;
    public username: string;
    public password: string;
    public brTelefona: string;
    public email: string;
    public status: string | null;
    public token: string | null;
    

    constructor(id: number, ime: string, prezime: string, username: string, email: string, brTelefona: string, password: string)
    {
        this.id = id;
        this.ime = ime;
        this.prezime = prezime;
        this.username = username;
        this.password = password;
        this.brTelefona = brTelefona;
        this.email = email;
        this.uloga = "korisnik";
        
        this.status = null;
        this.token = "";
    }

}

File: wlada-si.21.19.getout-e707bbd07835/Aplikacija/SWEproj/backend/GETOut/Models/Korisnik.cs
using System.ComponentModel.DataAnnotations;

namespace GETOut.Models
{
    public class Korisnik
    {
        public string Uloga { get; set; }

        public string Id { get; set; }

        public string Token { get; set; }

        public string ValidTo { get; set; }

        [Required(ErrorMessage = "Polje ime je obavezno")]
        public string Ime { get; set; }

        [Required(ErrorMessage = "Polje prezime je obavezno")]
        public string Prezime { get; set; }

        [Required(ErrorMessage = "Polje username je obavezno")]
        public string Username { get; set; }

        [Required(ErrorMessage = "Polje lozinka je obavezno!")]

        public string Password { get; set; }

        [Required(ErrorMessage = "Polje broj telefona je obavezno")]

        public string BrTelefona { get; set; }

        [Required(ErrorMessage = "Polje email je obavezno!")]
        public string Email { get; set; }
    }
}
